header.php

```diff
@@ -7,17 +7,27 @@
     <?php wp_head(); ?>
 </head>
 <body <?php body_class(); ?>>
-<header>
-    <div class="container">
-        <h1><a href="<?php echo esc_url(home_url('/')); ?>"><?php bloginfo('name'); ?></a></h1>
-        <nav>
-            <?php
-            wp_nav_menu(array(
-                'theme_location' => 'primary-menu',
-                'container' => false,
-                'menu_class' => 'nav-menu',
-            ));
-            ?>
-        </nav>
-    </div>
-</header>
\ No newline at end of file
+    <header class="header">
+        <div class="header__container">
+            <a href="<?php echo esc_url(home_url('/')); ?>" class="header__branding">
+                EZEBLaMER
+            </a>
+                <nav class="header__navigation">
+                    <a href="<?php echo esc_url(home_url('/')); ?>" 
+                        class="nav-link <?php echo is_front_page() ? 'active' : ''; ?>">
+                        <? echo __('Inicio', 'aprendiendoezequiel'); ?>
+                    </a>
+                    <a href="<?php echo esc_url(home_url('/acerca-de-mi')); ?>" 
+                        class="nav-link <?php echo is_page('acerca-de-mi') ? 'active' : ''; ?>">
+                        <? echo __('Acerca de mi', 'aprendiendoezequiel'); ?>
+                    </a>
+                    <a href="<?php echo esc_url(home_url('/proyectos')); ?>" 
+                    class="nav-link <?php echo is_page('proyectos') ? 'active' : ''; ?>">
+                    <? echo __('Proyectos', 'aprendiendoezequiel'); ?>
+                    </a>
+                </nav>
+                <a href="<?php echo esc_url(home_url('/contacto')); ?>" class="header__contact">
+                    Contáctame
+                </a>
```

Comment:
Esta desfasada la tabulacion aca
```suggestion
            <nav class="header__navigation">
                <a href="<?php echo esc_url(home_url('/')); ?>" 
                    class="nav-link <?php echo is_front_page() ? 'active' : ''; ?>">
                    <? echo __('Inicio', 'aprendiendoezequiel'); ?>
                </a>
                <a href="<?php echo esc_url(home_url('/acerca-de-mi')); ?>" 
                    class="nav-link <?php echo is_page('acerca-de-mi') ? 'active' : ''; ?>">
                    <? echo __('Acerca de mi', 'aprendiendoezequiel'); ?>
                </a>
                <a href="<?php echo esc_url(home_url('/proyectos')); ?>" 
                    class="nav-link <?php echo is_page('proyectos') ? 'active' : ''; ?>">
                    <? echo __('Proyectos', 'aprendiendoezequiel'); ?>
                </a>
            </nav>
            <a href="<?php echo esc_url(home_url('/contacto')); ?>" class="header__contact">
                Contáctame
            </a>
```